docs/kh2/file/type/preferences.md

```diff
@@ -0,0 +1,289 @@
+| 64 | float | Attack JV0
+| 68 | float | Attack First V0
+| 72 | float | Attack Combo V0
+| 76 | float | Attack Finish V0
+| 80 | float | Blow Recov H
+| 84 | float | Blow Recov V
+| 88 | float | Blow Recov Time
+| 92 | float | Auto Lock On Range
+| 96 | float | Auto Lock On Min H
+| 100 | float | Auto Lock On Max H
+| 104 | float | Auto Lock On Time
+| 108 | float | Auto Lock On H Adjust
+| 112 | float | Auto Lock On Inner Adjust
+
+
+### Fmab
+
+Each pointer leads to a specific entry's offset.
+
+### Fmab Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer Count [uint]
+| 5 	 | Pointers [uint]
+| 5 	 | Fmab Entries
+
+### Fmab Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | High Jump Height
+| 4 | float | Air Dodge Height
+| 8 | float | Air Dodge Speed
+| 12 | float | Air Slide Time
+| 16 | float | Air Slide Speed
+| 20 | float | Air Slide Brake
+| 24 | float | Air Slide Stop Brake
+| 28 | float | Air Slide Star Time
+| 32 | float | Glide Speed
+| 36 | float | Glide Fall Ratio
+| 40 | float | Glide Fall Height
+| 44 | float | Glide Fall Max
+| 48 | float | Glide Acceleration
+| 52 | float | Glide Start Height
+| 56 | float | Glide End Height
+| 60 | float | Glide Turn Speed
+| 64 | float | Dodge Roll Star Time
+
+### Prty
+
+Each pointer leads to a specific entry's offset.
+
+### Prty Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer Count [uint]
+| 70 	 | Pointers [uint]
+| 5 	 | Prty Entries
+
+### Prty Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | Walk Speed
+| 4 | float | Run Speed
+| 8 | float | Jump Height
+| 12 | float | Turn Speed
+| 16 | float | Hang Height
+| 20 | float | Hang Margin
+| 24 | float | Stun Time
+| 28 | float | Mp Drive
+| 32 | float | Up Down Speed
+| 36 | float | Dash Speed
+| 40 | float | Acceleration
+| 44 | float | Brake
+| 48 | float | Subjective Offset
+
+### Sstm
+
+Each pointer leads to a specific entry's offset.
+
+### Sstm Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer Count [uint]
+| 1 	 | Pointers [uint]
+| 1 	 | Sstm Entries
+
+### Sstm Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | Ceiling Stop
+| 4 | float | Ceiling Disable Command Time
+| 8 | float | Hang Range H
+| 12 | float | Hang Range L
+| 16 | float | Hang Range XZ
+| 20 | float | Fall Max
+| 24 | float | Blow Brake XZ
+| 28 | float | Blow Min XZ
+| 32 | float | Blow Brake Up
+| 36 | float | Blow Up
+| 40 | float | Blow Speed
+| 44 | float | Blow To Hit Back
+| 48 | float | Hit Back
+| 52 | float | Hit Back Small
+| 56 | float | Hit Back To Jump
+| 60 | float | Fly Blow Brake
+| 64 | float | Fly Blow Stop
+| 68 | float | Fly Blow Up Adjust
+| 72 | float | Magic Jump
+| 76 | float | Lock On Range
+| 80 | float | Lock On Release Range
+| 84 | float | Stun Recov
```

Comment:
Please use "Recovery" instead of "Recov"

Reply:
Extended.